Admin-app/src/components/AdminList/index.js

```diff
@@ -0,0 +1,178 @@
+/**
```

Comment:
Maybe for disabled accounts, they should be shown as greyed out (or less bright).

Reply:
Done